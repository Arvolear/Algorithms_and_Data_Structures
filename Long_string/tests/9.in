69186409105930189191999731776914122794624419
485696586327295