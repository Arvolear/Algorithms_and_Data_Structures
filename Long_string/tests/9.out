69186409105930189191999731777399819380951714
69186409105930189191999731776428426208297124
33603602722993971013213789508558455268458967280600133216605
142447797768353548240519148317